deidentification/index.md

```diff
@@ -0,0 +1,8 @@
+## De-identification
+
+The process of de-identification is removing identifiers from a dataset, such cities, dates, names, ID's.
+For a comprehensive overview of what de-identification is, see the privacy handbook [link needed].
```

Comment:
```suggestion
For a comprehensive overview of what de-identification is and how it can be applied to research data, please refer to the [Data Privacy Handbook](https://utrechtuniversity.github.io/dataprivacyhandbook/pseudonymisation-anonymisation.html){target="_blank"}.
```